corehq/apps/hqadmin/views.py

```diff
@@ -1190,3 +1191,18 @@ def top_five_projects_by_country(request):
                     .size(5).run().hits)
         data = {country: projects}
     return json_response(data)
+
+
+class WebUserDataView(View):
+    urlname = 'web_user_data'
+
+    def dispatch(self, request, *args, **kwargs):
+        return super(WebUserDataView, self).dispatch(request, domain='dummy', *args, **kwargs)
```

Comment:
the auth decorators require a domain argument, but dont need it to validate. this is kind of an ugly hack but it seems harmless